docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,88 @@
+"""
+Coriolis parameters across the world
+====================================
+
+This code computes the Coriolis parameter for an area and stores it in a cube.
+
+It creates and plots a map showing the Coriolis parameters, overlaid on top of
+an Orthographic projection of the globe.
+
+"""
+
+import cartopy.crs as ccrs
+import iris
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 200
+    longitudes = np.linspace(-180.0, 180.0, resolution)
+    latitudes = np.linspace(-90.0, 90.0, resolution)
+    lon2d, lat2d = np.meshgrid(longitudes, latitudes)
+
+    # Omega is the Earth's rotation rate, expressed in radians per second
+    omega = 7.29e-5
+
+    # Now to create our data
```

Comment:
Can we have a comment that states what the data are instead? Something along the lines of

```
Compute the Corilois parameter from the latitude values
```